System/Posix/IO/Common.hsc

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE CApiFFI #-}
 {-# LANGUAGE NondecreasingIndentation #-}
 {-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE Safe #-}
+{-# LANGUAGE Trustworthy #-}
```

Comment:
No, it's not essential. I had to remove the `Safe` pragma because the imported modules of bytestring are not safe.
Likewise modules marked as safe cannot depend on this module anymore, but I am not sure if that regression is a show stopper.